oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -218,9 +219,11 @@ def _build_sas_logical_jbods(server_profile_template, storage_blocks):
             server_profile_template)
 
         if storage_blocks and not controller:
-            raise OneViewRedfishError(
-                "The Server Profile Template should have a valid "
-                "storage controller to use the Storage Resource Blocks passed")
+            raise OneViewRedfishException(
+                "The Server Profile Template should have a valid storage "
+                "controller to use the Storage Resource Blocks passed",
+                status.HTTP_403_FORBIDDEN
```

Comment:
I think the api should not know about the HTTP status. It should just raise a exception, with some message, but doesn't specifying some HTTP status, the blueprint is who should catch the exception and return to the client the HTTP status properly.